webui/react/src/pages/F_ExpList/glide-table/MultiSortMenu.module.scss

```diff
@@ -0,0 +1,27 @@
+.base {
+  display: flex;
+  flex-direction: column;
+  gap: 8px;
+  max-height: 500px;
```

Comment:
super nit: 500px seems a little big. I might go for 400? That still fits 8 full sort rows at once, which seems like more than anyone should need.

Reply:
i'll defer to @alexjdetermined here.